src/components/Calculator.css

```diff
@@ -0,0 +1,34 @@
+#calculator {
+  background: green;
+  width: 200px;
+  height: 200px;
+}
+
+#calc-buttons {
+  display: flex;
+  flex-direction: column;
+}
+
+#display-calc {
+  height: 36px;
+  width: 100%;
+  background-color: grey;
+  text-align: right;
+  color: white;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+}
+
+#calc-buttons button {
+  padding: 8px 0;
+  flex-grow: 1;
+  width: 25%;
+}
+
+#calc-buttons .double-button {
+  width: 50%;
```

Comment:
- [ ] Make sure the color would be the same as the given design. Currently, you have different button color.

### Your calculator

![Screenshot from 2021-10-21 02-20-09](https://user-images.githubusercontent.com/22434660/138170305-a6f16e6e-19c0-4e81-9f92-c30e130bb86d.png)

### GIven calculator

![Screenshot from 2021-10-21 02-20-23](https://user-images.githubusercontent.com/22434660/138170354-1731a9f0-0258-43bb-a48a-ff83ac486929.png)